src/operator/nn/convolution.cc

```diff
@@ -99,6 +99,8 @@ static bool ConvolutionShape(const nnvm::NodeAttrs& attrs,
     // 1d conv
     CHECK_EQ(dshp.ndim(), 3U) << "Input data should be 3D in batch-num_filter-x";
     Shape<3> dshape = ConvertLayout(dshp.get<3>(), param_.layout.value(), kNCW);
+    CHECK_NE(param_.num_group, 0U) \
+      << "num_group must be non-zero";
```

Comment:
- the `num_group` argument can't be negative either so it would be better to check for `CHECK_GT(..., 0)`
- It would be better to report the received number.

Reply:
Thanks for your advice. We can use the following coding.
```
    CHECK_GT(param_.num_group, 0U) \
      << "Range only supports num_group > 0, received "<<param_.num_group;
```
Should I change the coding and commit it?

Reply:
Yes looks good. Thanks